internal/service/kendra/index.go

```diff
@@ -0,0 +1,1036 @@
+		},
+		CustomizeDiff: verify.SetTagsDiff,
+		Schema: map[string]*schema.Schema{
+			"arn": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"capacity_units": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"query_capacity_units": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+						"storage_capacity_units": {
+							Type:     schema.TypeInt,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"created_at": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"description": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validation.StringLenBetween(1, 1000),
+			},
+			"document_metadata_configuration_updates": {
+				Type:     schema.TypeSet,
+				Computed: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"relevance": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"duration": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"freshness": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"importance": {
+										Type:     schema.TypeInt,
+										Computed: true,
+									},
+									"rank_order": {
+										Type:     schema.TypeString,
+										Computed: true,
+									},
+									"values_importance_map": {
+										Type:     schema.TypeMap,
+										Computed: true,
+										Elem:     &schema.Schema{Type: schema.TypeInt},
+									},
+								},
+							},
+						},
+						"search": {
+							Type:     schema.TypeList,
+							Computed: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"displayable": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"facetable": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"searchable": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+									"sortable": {
+										Type:     schema.TypeBool,
+										Computed: true,
+									},
+								},
+							},
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+					},
+				},
+			},
+			"edition": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ForceNew:     true,
+				ValidateFunc: validation.StringInSlice(indexEditionValues(types.IndexEdition("").Values()...), false),
+				DiffSuppressFunc: func(k, old, new string, d *schema.ResourceData) bool {
+					// API returns "ENTERPRISE_EDITION" by default.
+					if old == string(types.IndexEditionEnterpriseEdition) && new == "" {
+						return true
+					}
+
+					return old == new
+				},
```

Comment:
maybe consider adding `Default:  string(types.IndexEditionEnterpriseEdition),` here instead?

Reply:
Resolved in 22c934249a7774416a04803e8938d509c82c192f